ios/Nodes/REAAlwaysNode.m

```diff
@@ -0,0 +1,33 @@
+#import "REAAlwaysNode.h"
+#import "REANodesManager.h"
+#import "REAStyleNode.h"
+#import "REAModule.h"
+#import <React/RCTLog.h>
+#import <React/RCTUIManager.h>
+
+@implementation REAAlwaysNode
+{
+  NSNumber * _nodesToBeEvaluated;
+}
+
+- (instancetype)initWithID:(REANodeID)nodeID config:(NSDictionary<NSString *,id> *)config
+{
+    if ((self = [super initWithID:nodeID config:config])) {
+        _nodesToBeEvaluated = config[@"what"];
+    }
+    return self;
+}
+
+- (id)value
```

Comment:
I think you should override `evaluate` here instead of `value`

`value` is a method that should not be overwritten (although there is no way in objc to prevent that). It calls into `evaluate` and memoizes the result so that it is guaranteed we "evaluate" at most once per frame